src/style.css

```diff
@@ -63,15 +102,41 @@ table {
   width: 50%;
 }
 
+.add-submit {
+  display: flex;
+  justify-content: flex-end;
+}
+
 .submit-btn {
-  box-shadow: 3px 3px rgb(219, 214, 214);
-  margin: 0 auto;
+  box-shadow: 3px 3px rgb(192, 187, 187);
   display: block;
   padding: 9px;
   background: none;
   border: none;
   font-size: 16px;
   font-weight: 500;
-  color: #666;
+  color: rgb(102, 97, 97);
   cursor: pointer;
+  margin-top: 10px;
+}
+
+.submit-btn:hover {
+  background-color: rgb(196, 180, 212);
+}
+
+#user-name {
+  margin-top: 23px;
+  margin-bottom: 5px;
+  width: 100%;
+  color: #fff;
+  padding: 8px 10px;
+  border: 1px solid #333;
+}
+
+#user-score {
+  margin-top: 9px;
+  width: 100%;
+  color: #fff;
```

Comment:
- [optional] Kindly take note that the `text color` and `background color` are the same, kindly change any of the two colors for readability purpose. 👍